test/lib/variables.test.ts

```diff
@@ -18,3 +18,6 @@ test('.Net C# project with variable is parsed', async (t) => {
   t.equal(depTree.dependenciesWithUnknownVersions!.length, 4);
 });
 
+test('props parser uses .props to resolve package versions referenced in *proj files', async (t) => {
+  const manifestFileContents = fs.readFileSync(`${__dirname}/../fixtures/dotnet-with-props/example.fsproj`, 'utf-8');
```

Comment:
what did we intend to test here?

Reply:
this can be PR 2, we wanted to test that if passed  in the PropsMap is used for processing the manifest